packages/process/_test.pony

```diff
@@ -92,6 +93,58 @@ class iso _TestFileExec is UnitTest
   fun timed_out(h: TestHelper) =>
     h.complete(false)
 
+class iso _TestExpect is UnitTest
+  fun name(): String =>
+    "process/Expect"
+
+  fun apply(h: TestHelper) =>
```

Comment:
I dont understand what is going on with this test.

i'm unclear how it is testing. 

we set expect to 4
then write three characters "one"
then expect to 0

for anything that is expecting 4, it shouldnt every be called.
so "one" should never happen. it should be buffered waiting for 1 more byte to be buffered.

I would expect a test to do something like write "hello there theo!!"

then do 

expect 2 and get

"he"

expect 4 and get

"llo "

expect 2 and get

"th"

expect 4 and get

"ere "

expect 2 and get 

"th"

expect 4 and get 

"eo!!"